src/main/java/seedu/address/model/person/Uid.java

```diff
@@ -0,0 +1,55 @@
+package seedu.address.model.person;
+
+import java.util.UUID;
+/**
+ * Uid that uniquely defines a person.
+ * Uid is an unmodifiable field that is created every time with the creation of a new Person.
+ */
+public class Uid {
+    public static final String MESSAGE_CONSTRAINTS = "Uid should be a string parsed from java UUID object";
+    public final String value;
+
+    /**
+     * constructor that creates an Uid Object.
```

Comment:
Change to uppercase and small grammar change

```suggestion
     * Constructor that creates a Uid Object.
```